docs/content/introduction/concepts/_index.md

```diff
@@ -35,3 +35,11 @@ A stage is composed of several pipelines. [See Workflow documentation]({{< relre
 
 ![Workflow](/images/introduction.concept.workflow.png)
 
+### A Project
+
+A project contains workflows, applications, pipelines and environments and can be linked to platforms
```

Comment:
A project contains workflows, applications, pipelines and environments. Each project can be linked to platforms

Reply:
do we talk about link to repositories managers ?

Reply:
A project represents an organisation, it contains...

Reply:
done